### ISSUE TITLE
Add Deploy Step causes crash

### ISSUE THREAD

**Issue by sevenbitbyte** [ros-industrial/ros_qtc_plugin#119]
Anytime I try to add a deploy step in the Run Settings accessed from the project `Build & Run` menu the UI locks and then crashes if I create a deploy step of type `CatkinMake Step` or `CatkinTools Step`. I tried this with a catkin tools workspace as well as a traditional catkin make setup.

I have had lots of different Qt versions installed so maybe I missed a setup or cleanup step thats causing my trouble

```
$ qtchooser -print-env
QT_SELECT="default"
QTTOOLDIR="/opt/qt57/bin"
QTLIBDIR="/opt/qt57/lib"
```

A possibly unrelated observation. When I use the play button to run the application it is started and running but the `Application Output` tab shows as if the application was never started and console output is detached. When I run in debug mode I get the messages `No executable specified. Debugging has finished` but once again the application is left running but with detached output.

**Comment by Levi-Armstrong** [comment]
Thank you for your feedback. I have not tested this, but I will look into it.

**Comment by kjeremy** [comment]
I'm seeing this too with CatkinTools Step.

**Comment by Levi-Armstrong** [comment]
This should be resolved by #152
